src/main/java/Seat.java

```diff
@@ -0,0 +1,30 @@
+public class Seat {
+    int seatNumber;
```

Comment:
zawsze zaczynaj od "private".

Reply:
private int seatNumber;

dopiero gdy na prawdę masz sensowny powod, aby było inaczej - zmieniasz to